.github/workflows/gha_generate_readme_and_issuetemp.yml

```diff
@@ -19,28 +19,27 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v3
 
-      - name: Setup Conda
-        uses: s-weigand/setup-conda@v1
+      - name: Setup Micromamba ${{ matrix.python-version }}
+        uses: mamba-org/setup-micromamba@v1
```

Comment:
The GHA above is old a a bit clunky. Using micromamba as the conda provider is not only faster for CIs but the "recommended way" for GitHub Actions.